tru_tags.php

```diff
@@ -849,12 +843,6 @@ function tru_tags_admin_redirect_tag($lefttag, $righttag) {
 		return 'Please enter a value in both fields';
 	}
 
-	if (!safe_query('create table if not exists ' . safe_pfx('tru_tags_redirects').
-		' (lefttag varchar(255) primary key, '.
-		'righttag varchar(255) not null)')) {
-		return 'Serious error - unable to create the ' . safe_pfx('tru_tags_redirects') . ' table.';
-	}
-
 	if (safe_insert('tru_tags_redirects', 'lefttag="'.$lefttag.'",righttag="'.$righttag.'"')) {
```

Comment:
I think that everything is in order here. Checking and creating a table occurs when you open the admin tab.